app/src/main/java/com/devshish/internship/data/db/RoomSongDao.kt

```diff
@@ -17,4 +17,7 @@ interface RoomSongDao {
 
     @Query("SELECT * FROM lyrics WHERE title = :title AND artist = :artist")
     suspend fun getStoredSong(title: String, artist: String): RoomSong?
+
+    @Query("SELECT EXISTS(SELECT * FROM lyrics WHERE title = :title AND artist = :artist)")
+    suspend fun isSongStored(title: String, artist: String): Boolean
```

Comment:
You can instead of creating new method - just use @Index in Room for these 2 fields 
And change this thing for Flow